pages/home/content.json

```diff
@@ -22,6 +22,31 @@
 	"clientsFullList": "Nasa - Bit of text about nasa and what it was like to go into space with them, FedEx, Starbucks, Ikea, and many more ...",
 	"clientsLink": "Full List of Clients",
 	"clientsLinkTarget": "#clients",
+	"team": [
+		{
+			"name": "Stuart Everate",
```

Comment:
Pls fix the typo in Stu's name. It's the kind of tiny thing that subconsciously affects how the client will feel about the site when they see it.